### PR TITLE
Update settings.php

### PR DESCRIPTION
**Pull request by kostefun**
Ru update

### CODE REVIEW

File: resources/lang/ru/settings.php [hunk 1]
```diff
@@ -42,21 +42,21 @@
...
+    'page_draft_color' => 'Цвет страницы черновика',
 
     // Registration Settings
     'reg_settings' => 'Настройки регистрации',
     'reg_enable' => 'Разрешить регистрацию',
     'reg_enable_toggle' => 'Разрешить регистрацию',
     'reg_enable_desc' => 'Если регистрация разрешена, пользователь сможет зарегистрироваться в системе самостоятельно. При регистрации назначается роль пользователя по умолчанию',
     'reg_default_role' => 'Роль пользователя по умолчанию после регистрации',
-    'reg_enable_external_warning' => 'The option above is ignored while external LDAP or SAML authentication is active. User accounts for non-existing members will be auto-created if authentication, against the external system in use, is successful.',
+    'reg_enable_external_warning' => 'Если используется LDAP или SAMP аутентификация, то параметр выше игнорируется. Аккаунты для несуществующих пользователей будут автоматически созданы если аутентификация на внешней системе будет успешна.',
```
Comment (MikeyMJCO) [inline resources/lang/ru/settings.php]:
```suggestion
    'reg_enable_external_warning' => 'Если используется LDAP или SAMP аутентификация, то параметр выше игнорируется. Аккаунты для несуществующих пользователей будут автоматически созданы если аутентификация на внешней системе будет успешна.',
```

Typo - SAMP should be SAML?

Comment (kostefun) [reply resources/lang/ru/settings.php]: ok

**Comment by MikeyMJCO** [comment]
@kostefun doing some digging on this (newish to BookStack!) you should do the translations via Crowdin here: https://crowdin.com/project/bookstack/ru# they will then be "synced" into BookStack for the next release I believe!

**Comment by ssddanbrown** [comment]
@kostefun Thanks for offering these translations but I see you have been since active on crowdin and made the updates there, Thank you very much. Since this pull request is now redundant (Tried to merge it but ended up with no conflicts after resolution) I'll close this.
Your changes via crowdin have been merged and will be part of the next patch release.

Thanks @MikeyMJCO for highlighting the crowdin system.

**Pull request closed without merging**